README.md

```diff
@@ -1,123 +1,15 @@
-# Differ.rs
-[![License](https://img.shields.io/badge/license-MIT%20%26%20Apache%202.0-green)](#license)
-[![CI](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml)
-[![Security audit](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml)
-> warning: **Differ.rs is currently experimental**
-This crate provides edit distance, delta vectors between 2 words, and lets you apply delta vectors in order to transform words.
+# differ.rs
 
-## Install
-```shell
-cargo add differ-rs
-```
-or, simply add the following string to your Cargo.toml:
+A natural language processing library, written in Rust.
 
-```toml
-differ-rs = "0.0.0"
-```
+# *Brief overview*
+### Crates
+ - [x]  Differ:
+   - [x]  `apply_diff`: Allows users to apply delta vectors in order to transform a words.
+   - [x]  `extra_traits`: all `struct`s implemented in `differ-rs` are `HammingDistance` and `LevenshteinDistance`. Each Struct implements the `diff` and `distance` methods.
```

Comment:
Since the API details of a library can change, the "root" README should just be the main description of the crate (to make it easier, it should just be the same description you wrote for it in it's `Cargo.toml`) You also mentioned that it should still be called `bio_diff`, so we should call it that too:

```markdown
 - [x] [`differ`](./src/crates/differ): <description of crate>
 - [ ] `bio_diff`: <description of crate>
 ```
 
 And then, we can add a link to `bio_diff` once it's actually available.